examples/01-Detailed-Examples/05-mesh-exploration.py

```diff
@@ -0,0 +1,203 @@
+"""
+.. _ref_mesh_exploration_example:
+
+Explore the mesh
+================
+In this script a static simulation is used as an example to show how to
+query mesh information such as connectivity, element IDs, element types and so on.
+"""
+
+###############################################################################
+# Perform required imports
+# ------------------------
+# Perform required imports. # This example uses a supplied file that you can
+# get by importing the DPF ``examples`` package.
+
+from ansys.dpf import post
```

Comment:
as we discussed @ansys-akarcher, in my opinion this is not really an example, it's more an api display, if you could move these to the mesh docstrings, it would be great

Reply:
@cbellot000 yes, I just added examples in dosctrings of post.Mesh
Will be updating the example to make a proper one